src/util.js

```diff
@@ -311,3 +311,16 @@ function setLocEnd(node, index) {
   }
 }
 util.setLocEnd = setLocEnd;
+
+// http://stackoverflow.com/a/7124052
+function htmlEscapeInsideDoubleQuote(str) {
+  return str
+    .replace(/&/g, '&amp;')
```

Comment:
It does?! I'm confused! In JavaScript `"&amp;"` is just a normal string value, it doesn't mean anything until it touches the DOM right?